src/Wizard/Readline/ReadlinePlatform.ts

```diff
@@ -154,6 +160,26 @@ export class ReadlinePlatform implements Platform {
     );
   }
 
+  private async processTraceroute(options?: StartOptions): Promise<void> {
+    console.log(
+      `Traceroute to your INTERNAL (local) target application.${EOL}`
+    );
+    console.log(`Starting INTERNAL communication diagnostics:${EOL}`);
+
+    await this.process(
+      `Trying to reach ${options.traceroute}`,
```

Comment:
In case traceroute, `process()` is useless. You just need to display its output without `Trying to reach ${options.traceroute}... Success | Failed`